### DISCUSSION TITLE
Is there a command to add quotations?

### DISCUSSION THREAD

**Discussion by galaxea** [openai/whisper#844] [Q&A] [answered]
Whisper has fantastic accuracy, including helping me not to even have to say the usual 'comma','period', etc  commands that software such as Nuance makes me use. However, is there a command to utilize quotation marks? I rough draft forms of fiction using a recorder and then put it through whisper. Dialogue is my only real obstacle I'm trying to overcome. 

Any help would be appreciated.

**Comment by ryanheise** [comment, marked as answer]
Whisper can infer quotes and other symbols, but they are suppressed by default. To prevent these from being suppressed, you can pass the `--suppress_tokens ''` option on the command line or the `suppress_tokens=''` option in Python, which specifies that nothing should be suppressed.

**Reply by galaxea** [reply]
--suppress_tokens 0 still keeps punctuations but turns open and close quote to " " marks, which is what I needed. Thanks! Now to see if I figure how to add it to the speech translate GUI.

**Reply by NormalFall0** [reply]
Hello guys I am not able to use this supresstoehen, how are you doing it? I got this error:
![image](https://github.com/openai/whisper/assets/169382007/9e18de01-b92e-491f-a137-e8cceea0dfff)

**Reply by NormalFall0** [reply]
I tried with '0' and it seems to work; althought It seems you guys dont put anything inside it
